components/camel-salesforce/it/resources/CamelSalesforceIntegrationTests.connectedApp

```diff
@@ -21,8 +21,8 @@
     <oauthConfig>
         <callbackUrl>https://login.salesforce.com/services/oauth2/success</callbackUrl>
         <certificate>MIIC1TCCAb2gAwIBAgIEM3ZMGjANBgkqhkiG9w0BAQsFADAaMRgwFgYDVQQDEw9TYWxlc2ZvcmNlIHRlc3QwIBcNMTcwMzE0MjIxMjU0WhgPMjExNzAyMTgyMjEyNTRaMBoxGDAWBgNVBAMTD1NhbGVzZm9yY2UgdGVzdDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAMbZinVCZerzZvyyQlfE6nMpEQRfVsjpcfT01UTG/bwzWorP7YRpGkDW7Q4eu6IPrHtohkhM3JtsSVka5jfS1iEguMXLdNkEyMjMiBrWJeyGfcISF1yazgqLxxwcwGjMn3C9xV5tBxiRSqtMrV1iRx3fxmLue1UnZjSyUaG+Vi+FcKxqre5ixApeDCZHLONxBy3mjWK4GIeBBbUqQqy3LNrT6B34WdNX8vTslpTKOlmLyycEI/Rx+A4lNaultrJHdnRhGBrPwYinlV/wdramaQFL+ltqxQw6A5FBg9VjDEEd8JGv6OZ7aA/3mvjTbX5MaLj+aLJQyT2+XzDSAtDgOhMCAwEAAaMhMB8wHQYDVR0OBBYEFDM0jGtarY5uTVLThBEnXMUqDlyDMA0GCSqGSIb3DQEBCwUAA4IBAQCJtIsP5foO84sNwVMGwPiaoQUPE+cN73Tg63FSef36FEei6B3UnsFZnlcz9WGIL0G4PUjQro5z7/gMAHGehPQjoUEzKihdoNncbvKa9Yfuw4eO98YIp2t5OnOGQpDQWN9JO0On0tu+4rsgkJn8fZtzTPViS2Vhssi65DpWYydKWvEbbYw6oiPMH9T2mQy11lxSwNBngBy4ZF4hCyEVpvSe6Mr2q+jxgjCwMAqX+XofmEpfHaJXGy8lgtzow0oT8X4ymsV1JR0erw0dsbch9pTsG7YXcNFIEm50DctQEikoWxkAvIsZIHgujWkrjQlEsXZp/MS4OuNj6yWWlCaK/QM3</certificate>
-        <consumerKey>${salesforce.client.id}</consumerKey>
-        <consumerSecret>${salesforce.client.secret}</consumerSecret>
+        <!--consumerKey>${salesforce.client.id}</consumerKey>
+        <consumerSecret>${salesforce.client.secret}</consumerSecret-->
```

Comment:
Is there any reason to comment these two propreties? they are not used in the test?

Reply:
Hi @zhfeng ,

No, they are not used in the test, the test will pick up the salesforce.client.id and salesforce.client.secret properties from test-salesforce-login.properties directly.

Plus, having consumerKey and consumerSecret in CamelSalesforceIntegrationTests.connectedApp actually breaks the latest version of ant migration tool(ant-salesforce.jar complains those and failed). Only removing those can setup the salesforce server and run the integration test.

Cheers
Freeman

Reply:
So it should be better to remove them but not just comment. Or add some words to explain this reason.